doc/dev.rst

```diff
@@ -249,49 +260,139 @@ What kind of issue is it?
   starting work on it.
 
-:code:`duplicate`
-  Same as some other previously reported issue. In addition to this tag,
-  duplicates should refer to the other issue and be closed.
+Note that (1) the priority levels are indeed fuzzy, and we have no plans to
+make them more precise; (2) issues are not a priority queue, partly for this
+reason, i.e., we do work on lower-priority issues while higher-priority ones
+are still open; and (3) issues often move between priority levels. In
+particular, if you think we picked the wrong priority level, please say so.
+
+What part of Charliecloud is affected?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Choose *one or more components* from:
+
+:code:`runtime`
+  The container runtime itself; largely :code:`ch-run`.
+
+:code:`image`
+  Image building and interaction with image registries; largely
+  :code:`ch-image`. (Not to be confused with image management tasks done by
+  glue code.)
+
+:code:`glue`
+  Largely shell scripts in :code:`bin`.
+
+:code:`build`
+  Build system and packaging. (Not to be confused with image building.)
+
+:code:`doc`
+  Documentation.
 
-:code:`obsolete`
-  No longer relevant, moot, etc. Close.
+:code:`test`
+  Test suite.
 
-:code:`erroneous`
-  Not a Charliecloud issue; close. *Use caution when blaming a problem on user
-  error. Often (or usually) there is a documentation or usability bug that
-  caused the "user error".*
+:code:`misc`
+  Everything else. Do not combine with another component.
+
+Special considerations
+~~~~~~~~~~~~~~~~~~~~~~
+
+Choose *one or more extras* from:
+
+:code:`usability`
+  Affects usability of any part of Charliecloud, including documentation and
+  project organization.
+
+:code:`hpc`
+  Related specifically to HPC and HPC scaling considerations; e.g.,
+  interactions with job schedulers.
+
+:code:`uncertain`
+  Course of action is unclear. For example: is the feature a good idea,
+  what is a good approach to solve the bug, what additional information is
+  needed.
+
+Why was it closed?
+~~~~~~~~~~~~~~~~~~
+
+If the issue was resolved (i.e., bug fixed or enhancement/refactoring
+implemented), there is no disposition tag. Otherwise, to explain why not,
+choose *one disposition* from:
+
+:code:`discussion`
+  Converted to a discussion. The most common use is when someone asks a
+  question rather than making a request for some change.
+
+:code:`duplicate`
+  Same as some other issue. In addition to this tag, duplicates should refer
+  to the other issue in a comment. Of the duplicates, the better one should
+  stay open (e.g., clearer reproduction steps); if they are roughly equal in
+  quality, the older one should stay open.
+
+:code:`jurisdiction`
+  The issue is not something we can resolve. Typically problems with other
+  software or problems with containers in general that we can't work around.
+  *Use caution when blaming a problem on user error. Often (or usually) there
+  is a documentation or usability bug that caused the "user error".*
```

Comment:
Minor thing but I'm not sure I like `jurisdiction` although I'm having a tough time articulating why exactly. Perhaps `scope` or `external` instead?

Reply:
Other less friendly alternatives I came up with:

* `notcharliecloud`
* `notourproblem`
* `yourmonkeys`

Reply:
So far I don't see anything I like better than jurisdiction. I agree it's a tad awkward. How much do folks dislike it?

Reply:
* `nothingtodo`
* `notactionable`
 
?

Reply:
I think `cantfix` is more friendly than `erroneous` and a bit more helpful than `jurisdiction`.

Reply:
I like `cantfix` from that new list.